File: python/liczby23.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  liczby23.py
#
#  
#  
def liczby2():
    ile = 0
    for i in range(1, 10):
        for j in range(10):
            if i != j:
                print("{}{} ".format(i, j), end="")
                ile = ile + 1
    return ile
    return 0


def main(args):
    print(liczby2())
    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/liczby23.cpp
/*
 * liczby23.cpp
 * 
 * 
 * 
 */


#include <iostream>
using namespace std;

int liczby2() {
    int ile = 0;
    for (int i = 0; i < 10; i++) {
        for (int j = 0; j < 10; j++) {
            if (i != j){
                cout << i << j << " ";
                }
            }
        }
        return ile;
    }

int main(int argc, char **argv)
{cout << "liczby dwucyfrowe" << liczby2();
	
	return 0;
}

